### DISCUSSION TITLE
Updating outdated Windows 11 on ARM on UTM M1 Mac

### DISCUSSION THREAD

**Discussion by Scaramanga89** [utmapp/UTM#4413] [UTM Help (macOS)]
I have had this running perfectly for months, but now I get a message that my version of Windows has expired, I should move to a new version. However, I cannot update within Windows as the device is not supported 

![Screenshot 2022-09-14 at 10 02 36](https://user-images.githubusercontent.com/98148862/190111249-b3ad8fe0-aaf7-4f45-bfc0-5e4299388936.png)

as it's an M1 Mac, so I cannot move to the Beta channel on Windows Insider. I cannot install from a mounted ISO as you cannot mount from any anything that's not NTFS formatted, and I cannot even mount from an external USB drive. Suggestions welcome, do I just need to create a new VM? I've also tried downloading and importing the ISO using OneDrive, but it also gives an error that it cannot be started properly.  Is there a way to move to the BETA channel, and it would update automatically for me, perhaps with the registry hack? 

![Screenshot 2022-09-14 at 11 06 33](https://user-images.githubusercontent.com/98148862/190111940-4dc8fe54-3643-4b52-8861-124215ca16b3.png)

**Comment by nfwatsoncom** [comment]
I also am getting this problem now.  I tried re-downloading the Windows 11 VHDX (?) file and installing, it's not updated either, I get a similar message on boot-up during install.

**Reply by Scaramanga89** [reply]
I reinstalled with the latest version of Windows 11 that was released last week and I still see the message. So I just ignored it. I also hacked the registry to enable the beta channel for updates, but that won't update either. Going to have to live with it unless I can make a VDHX out of an ISO.

**Comment by benguild** [comment]
Also getting this here. I tried following these steps, but it doesn't find the update:
https://www.stephenwagner.com/2022/11/09/how-to-force-windows-11-22h2-feature-update/

I'm currently creating a new ISO to see if it's possible to boot from that to update or reinstall, but I'd prefer to not have to set up the VM again from scratch.

**Comment by benguild** [comment]
So I used the ISO tools in the UTM instructions to make a new ISO, and then rebooted into system "CD boot" mode (using "Recovery options" in Settings on Windows) with the ISO mounted from the host machine.

You have to do the "Advanced" install (which moves all files and programs into "Windows.old") which kind of sucks, but you'll still have access to the data on disk to reinstall apps and transfer documents back over manually. For me, I don't use the VM for much, so I only had to do this for three different programs.
